lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/IMRUDriver.cs

```diff
@@ -115,11 +130,15 @@ internal sealed class IMRUDriver<TMapInput, TMapOutput, TResult> : IObserver<IDr
             _partitionDescriptorStack = new Stack<IPartitionDescriptor>();
             ConstructTaskIdAndPartitionDescriptorStack();
```

Comment:
You can return new Stack<IPartitionDescriptor> from ConstructTaskIdAndPartitionDescriptorStack(), then you don't need class variable _partitionDescriptorStack.